lib/resources/majors.tsx

```diff
@@ -2425,144 +2479,238 @@ const bsCS_New: Major = {
-            'requirements, for example to count as both Team and Software.</p>',
+            '<b>Team Requirement</b> <br /> ' +
+            'Select one course with Program of Study Tag CSCI-TEAM.',
           required_credits: 3,
           criteria: 'CSCI-TEAM[T]',
           double_count: [
-            '<b>Upper Level Undergraduate: </b><p>12 upper level CS credits in addition to the required Algorithms course</p>',
-            '<b>2 Upper Level Classifications:</b><p>At least one upper level course in two of these four different classification</p> ' +
-              'areas: Applications(CSCI-APPL), Systems(CSCI-SYST), Software(CSCI-SOFT) and Reasoning(CSCI-RSNG)',
+            '<b>Upper Level CS Credits</b> <br /> ' +
+              'At least 12 more upper level CS credits are required. ',
+            '<b>CS Electives</b> <br /> ' +
+              'Six additional credits of Computer Science are required.' +
+              'For an approved list of courses from other departments (maximum of 6 credits allowed), visit https://www.cs.jhu.edu/computer-science-other-courses-for-bs-degree/ ' +
+              'and create a custom course to satisfy this requirement.',
           ],
         },
       ],
     },
     {
-      name: 'Math',
-      required_credits: 16,
+      name: 'Computer Science Classifications',
+      required_credits: 6,
       min_credits_per_course: 3,
+      description:
+        'At least one course in two different classification areas (Applications, Reasoning, Software, Systems) must be chosen in addition to Theory (Algorithms).',
+      criteria: 'CSCI-APPL[T]^OR^CSCI-SOFT[T]^OR^CSCI-SYST[T]^OR^CSCI-RSNG[T]',
+      pathing: 2,
+      fine_requirements: [
+        {
+          description: '<b>Application</b>',
+          required_credits: 3,
+          criteria: 'CSCI-APPL[T]',
+          double_count: ['N/A'],
+        },
+        {
+          description: '<b>Software</b>',
+          required_credits: 3,
+          criteria: 'CSCI-SOFT[T]',
+          double_count: ['N/A'],
+        },
+        {
+          description: '<b>Systems</b>',
+          required_credits: 3,
+          criteria: 'CSCI-SYST[T]',
+          double_count: ['N/A'],
+        },
+        {
+          description: '<b>Reasoning</b>',
+          required_credits: 3,
+          criteria: 'CSCI-RSNG[T]',
+          double_count: ['N/A'],
+        },
+      ],
+    },
+    {
+      name: 'Mathematics',
+      required_credits: 8,
+      min_credits_per_course: 4,
       description:
         'All courses in this category must be from one of the two math departments on ' +
         'campus: Mathematics or Applied Math and Statistics. However, 553.171 Discrete Mathematics ' +
         'may not count towards these math requirements. Other than Calculus I and II, all the ' +
         'remaining courses must be 200-level or above.',
       criteria: 'AS Mathematics[D]^OR^EN Applied Mathematics & Statistics[D]',
```

Comment:
Criteria could be more specific so that prob stat doesn't end up here ^^

Reply:
Addressed